### PR TITLE
Add new-contributors.md

### PR DESCRIPTION
**Pull request by fabiand**
Small document to provide some introductionlinks to related projects.

### CODE REVIEW

File: docs/new-contributors.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Introduction
+
+This is a small guide of how to get started with Kubernetes, OpenShift Origins, and KubeVirt. 
+But let's start with how they relate to each other.
+
+*   **Kubernetes** is the upstream container orchestration engine, to run containers on a cluster
+*   **OpenShift** **Origin** is a friendly fork of Kubernetes with additional features, and configured to meet more enterprise requirements (i.e. RBAC authentication) than stock Kubernetes
```
Comment (berrange) [inline docs/new-contributors.md]: AFAIK, nothing in kubevirt relies on openshift specific features, so i'd rather not mention openshift at all - it'll just serve to confuse IMHO.

Comment (fabiand) [inline docs/new-contributors.md]: Hm, makes sense.  Especially if I consider i.e. tectonic, which is yet another K8s distrobution, then there is no particular reason why it must be mentioned here.

Removing it.

File: docs/new-contributors.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Let's start with the realtionship between the two important components:
```
Comment (mpolednik) [inline docs/new-contributors.md]: s/realtionship/relationship

File: docs/new-contributors.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+*   **Kubernetes** is the upstream container orchestration engine, to run containers on a cluster
```
Comment (mpolednik) [inline docs/new-contributors.md]: ... engine that runs containers in a cluster?

File: docs/new-contributors.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+*   **KubeVirt** is an add-on which is installed on-top of Kubernetes, to be able to add basic virtualization functionality to Kubernetes.
+
+Even that KubeVirt is an add-on to Kubernetes, both of them have things in common:
+
+*   Mostly written in Go Lang
```
Comment (mpolednik) [inline docs/new-contributors.md]: golang

File: docs/new-contributors.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+### [KubeVirt](https://github.com/kubevirt/) 
+
+*   Getting started  
+    *   [Demo](https://github.com/kubevirt/demo)
+*   Details
+
+## Topics
+
+### Go Lang
```
Comment (mpolednik) [inline docs/new-contributors.md]: golang

File: docs/new-contributors.md (hunk 3 above)
Comment (fabiand) [inline docs/new-contributors.md]: A bit cumbersome, rephrasing

**Comment by fabiand** [comment]
Thanks updated

File: docs/new-contributors.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Let's start with the relationship between the two important components:
+
+*   **Kubernetes** is a container orchestration system, and is used to run containers on a cluster
+*   **KubeVirt** is an add-on which is installed on-top of Kubernetes, to be able to add basic virtualization functionality to Kubernetes.
+
+Even that KubeVirt is an add-on to Kubernetes, both of them have things in common:
```
Comment (berrange) [inline docs/new-contributors.md]: Trivial typo s/that/though/

File: docs/new-contributors.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Let's start with the relationship between the two important components:
+
+*   **Kubernetes** is a container orchestration system, and is used to run containers on a cluster
+*   **KubeVirt** is an add-on which is installed on-top of Kubernetes, to be able to add basic virtualization functionality to Kubernetes.
```
Comment (berrange) [inline docs/new-contributors.md]: Where reasonable, try to keep line length below 80 chars. A few more cases below, but don't artificially break hyperlinks in the middle

File: docs/new-contributors.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+*   Getting started  
+
+    *   [Documentation - The Go Programming Language](https://golang.org/doc/ "https://golang.org/doc/") 
+    *   [Getting Started - The Go Programming Language](https://golang.org/doc/install "https://golang.org/doc/install") 
+
+Patterns
+
+*   [Introducing Operators: Putting Operational Knowledge into Software](https://coreos.com/blog/introducing-operators.html "https://coreos.com/blog/introducing-operators.html") 
+*   [Microservices](https://martinfowler.com/articles/microservices.html "https://martinfowler.com/articles/microservices.html") nice content by Martin Fowler
```
Comment (berrange) [inline docs/new-contributors.md]: Formatting in this block looks kind of odd.   'Getting started' is in a list bullet, but 'Patterns' is not ?!?

File: docs/new-contributors.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
+# Introduction
+
+Let's start with the relationship between the two important components:
+
+*   **Kubernetes** is a container orchestration system, and is used to run containers on a cluster
```
Comment (berrange) [inline docs/new-contributors.md]: Nit-pick - only need a single whitespace after the list bullet - repeated elsewhere in the file

Comment (fabiand) [inline docs/new-contributors.md]: Fixed

File: docs/new-contributors.md (hunk 7 above)
Comment (fabiand) [inline docs/new-contributors.md]: Yeah, that's tricky if the links get to long.
For now I shortened the lines, but even if the link exceeded the 80char limit, then I left it as it.

File: docs/new-contributors.md (hunk 6 above)
Comment (fabiand) [inline docs/new-contributors.md]: Fixed

File: docs/new-contributors.md (hunk 8 above)
Comment (fabiand) [inline docs/new-contributors.md]: Yeah, fixed it.

**Comment by fabiand** [comment]
Updated.

**Comment by fabiand** [comment]
Sigh - Now even with fixed indentation.

**Pull request merged**